piControlMain.c

```diff
@@ -456,7 +465,8 @@ static void __exit piControlCleanup(void)
 	if ((piDev_g.machine_type == REVPI_CORE ||
 	     piDev_g.machine_type == REVPI_CORE_SE||
 	     piDev_g.machine_type == REVPI_CONNECT_SE ||
-	     piDev_g.machine_type == REVPI_CONNECT) && isRunning())
+	     piDev_g.machine_type == REVPI_CONNECT ||
+	     piDev_g.machine_type == REVPI_CONNECT_4) && isRunning())
```

Comment:
Add another flag for these checks, e.g. `pibridge_supported`.

Reply:
Preferrably add these flags in separate commits at the front of the series. The Connect 4 bringup then merely consists of adding Connect 4 to the check which sets `pibridge_supported`.

Reply:
See https://github.com/RevolutionPi/piControl/pull/94/commits/e0ed45caf7bf42b8b0b4666838440a674901bc7c